segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerConfig.java

```diff
@@ -40,6 +40,7 @@
     public static final Property<Integer> MAX_CACHED_EXTENDED_ATTRIBUTE_COUNT = Property.named("extended.attribute.cached.count.max", 4096, "maxCachedExtendedAttributeCount");
     public static final Property<Integer> EVENT_PROCESSOR_ITERATION_DELAY_MS = Property.named("eventprocessor.iteration.delay.ms", 100);
     public static final Property<Integer> EVENT_PROCESSOR_OPERATION_TIMEOUT_MS = Property.named("eventprocessor.operation.timeout.ms", 5000);
+    public static final Property<Integer> TRANSIENT_SEGMENT_DELETE_TIMEOUT_MS = Property.named("segment.transient.delete.timeout.ms", 1000);
```

Comment:
I believe this is way too aggressive. If the segment store is busy, it's quite likely that requests may take longer than 1s to complete. I think 10 or 30 seconds would be a reasonable default.